source/_patterns/00-protons/_global.scss

```diff
@@ -5,10 +5,72 @@
 @import "uswds-theme-utilities";
 @import "uswds-theme-components";
 
+html,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+abbr,
+address,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+samp,
+small,
+strong,
+sub,
+sup,
+var,
+b,
+i,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
@slowbot Can we not do this here?  I need to import this file in the Drupal CSS/JS build, as mentioned in #100, which is that I need a file or two to import the dependencies (variables, colors, mixins, whatever), and which does not output any CSS. Adding CSS in here that will produce output here, means this will print twice (in courtyard css and atrium's css), and will likely cause cascade issues.  Maybe it would be better to have a `_base.scss` and `_global.scss`?

Reply:
Thanks!